chaosazure/vmss/records.py

```diff
@@ -0,0 +1,21 @@
+from calendar import timegm
+from datetime import datetime
+
+
+class Records:
+    elements = []
+
+    def __init__(self):
+        self.elements = []
+
+    def add(self, element: dict):
+        element['performed_at'] = timegm(datetime.utcnow().utctimetuple())
```

Comment:
I'm wondering if we need this as in the action there is an infor about duration, start time and end time.

In what situation do you think it would be useful?

Reply:
My thinking was that when you execute more than one operation - some may take longer, some may last shorter. The operations may vary  in their timepoints when finished. To be more accurate the records do collect the datetime of the performed action as well.

When you think further: it could be useful when you as a operator have the feeling that there is one machine that always takes a little bit longer and you're wondering why. This is going to analytics of your cluster.

Anyway I guess it's not hurting - in contrast can give some value back.